### ISSUE TITLE
Add `swup.say` method

### ISSUE THREAD

**Issue by hirasso** [swup/a11y-plugin#37]
**Describe the problem**

When using in-site app views that alter the URL but are being ignored by swup due to the use of `resolveUrl`, a11y plugin doesn't announce anything, since there is no swup visit happening.

**Describe the proposed solution**

It would be handy to be able to manually trigger announcements:

```js
import SwupA11yPlugin from "@swup/a11y-plugin";
const swup = new Swup({
  plugins: [
    new SwupA11yPlugin(),
  ],
});

// sometime later:
createHistoryRecord(newUrl);
swup.say?.(`Navigated to filter ${currentFilterName}`);
```

**How important is this feature to you?**

- [x] Nice to have
